website-frontend/src/components/Activities.js

```diff
@@ -1,6 +1,115 @@
+        <Divider />
+
+        <div className="row">
+          <div className="col-6">
+            <Card image={ dummypj1 } title='Landscape Research Project' date='Present'
+            excerpt='Purpose: This project aims to collect data about our surroundings and analyze the abc of xyz...
+            Field: Chemistry, Biology...' />
+          </div>
+          <div className="col-6">
+            <Card image={ dummypj2 } title='STEM Club Website' date='Past'
+            excerpt='Purpose: This project aims to create a professional identity for Fulbright STEM Club beyond Fanpage.
+            Field: Computer Science, Web Design.' />
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col-6">
+            <Card image={ dummypj3 } title='SQL Workshop Series' date='Upcoming'
+            excerpt='Purpose: This project aims to collect data about our surroundings and analyze the abc of xyz...
+            Field: Chemistry, Biology...' />
+          </div>
+
+          <div className="col-6">
+            <Card image={ dummypj4 } title='STEMazine' date='Past'
+            excerpt='Purpose: This project aims to create a professional identity for Fulbright STEM Club beyond Fanpage.
+            Field: Computer Science, Web Design' />
+          </div>
+        </div>
+      </div>
+
+      {/* EVENTS */}
+      <div>
+        <h3 className="text-primary">Events</h3>
+        <Divider />
+        <div className="row">
+          <div class="col-6">
+            <Card image={ dummy1 } title='Trivia Night' date='Dec 12, 2021' 
+            excerpt='Topic: On Dec 12, 2021, STEM Club hosted successfully Trivia Night to 
+            create a playground for STEM and non-STEM background students at Fulbright campus.'/>
+          </div>
+          <div class="col-6">
+            <Card image={ dummy2 } title='Sharing with NASA Intern' date='Feb 10, 2022' 
+            excerpt='Topic: During the talk, insights about being an intern in such an 
+            prestigious STEM organization, the pros and cons of being an data analyst 
+            intern at NASA'/>
+          </div>
+        </div>
+      </div>
+
+      {/* COMPETITION */}
+      <div>
+        <h3 className="text-primary">Competition</h3>
+        <Divider />
+        
+        <div className="row">
+          <div class="col-6">
+            <Card image={ dummycompetition1 } title='ICPC 2022' date='Mar 1st, 2022' 
+            excerpt='A 4-member team from STEM Club has participated in the competition.
+            The team has finished the challenge with a really promising result.'/>
+          </div>
+          <div class="col-6">
+            <Card image={ dummycompetition2 } title='STEM Hackathon' date='Jun 7th, 2021' 
+            excerpt='STEM Hackathon is a place for STEM enthusiasts to join and challenge 
+            themselves with STEM topics throughout 72 hours.'/>
+          </div>
+        </div>
+      </div>
```

Comment:
In the near future, all these fields (title, date, excerpt, image url) will be received from the server in JSON format. Right now, we can already mimic that by putting all those things in a JSON file (say, `src/data/mock/activities.json`). In that way, you can implement the logic to automatically render the rows and columns without knowing how many of them there are.

Please update this part accordingly.